docs/manual/developer_guide.adoc

```diff
@@ -1337,13 +1337,321 @@ Available low-level Jinja macros that can be used in Bash remediations:
+* Checks kernel command line arguments in GRUB 2 configuration.
+* Parameters:
+** *arg_name* - argument name, eg. `audit`
+** *arg_value* - argument value, eg. `'1'`
+* Languages: Ansible, Bash, OVAL
+
+kernel_module_disabled::
+* Checks if the given Linux kernel module is disabled.
+* Parameters:
+** *kernmodule* - name of the Linux kernel module, eg. `cramfs`
+* Languages: Ansible, Bash, OVAL
+
+mount::
+* Checks that a given mount point is located on a separate partition.
+* Parameters:
+** *mountpoint* - path to the mount point, eg. `/var/tmp`
+* Languages: Anaconda, OVAL
+
+mount_option::
+* Checks if a given partition is mounted with a specific option such as "nosuid".
+* Parameters:
+** *mountpoint* - mount point on the filesystem eg. `/dev/shm`
+** *mountoption* - mount option, eg. `nosuid`
+** *filesystem* - filesystem in `/etc/fstab`, eg. `tmpfs`. Used only in Bash remediation.
+** *type* - filesystem type. Used only in Bash remediation.
+** *mount_has_to_exist* - Used only in Bash remediation. Specifies if the *mountpoint* entry has to exist in `/etc/fstab`. If set to `yes` and the *mountpoint* entry is not present in `/etc/fstab` the Bash remediation terminates.
+* Languages: Anaconda, Ansible, Bash, OVAL
+
+mount_option_remote_filesystems::
+* Checks if all remote filesystems (NFS mounts in `/etc/fstab`) are mounted with a specific option.
+* Parameters:
+** *mountpoint* - always set to `remote_filesystems`
+** *mountoption* - mount option, eg. `nodev`
+** *filesystem* - filesystem of new mount point (used when adding new entry in `/etc/fstab`), eg. `tmpfs`. Used only in Bash remediation.
+* Languages: Ansible, Bash, OVAL
+
+mount_option_removable_partitions::
+* Checks if all removable media mounts are mounted with a specific option.
+* Parameters:
+** *mountpoint* - always set to `remote_filesystems`
+** *mountoption* - mount option, eg. `nodev`
+** *filesystem* - filesystem of new mount point (used when adding new entry in `/etc/fstab`), eg. `tmpfs`. Used only in Bash remediation.
+* Languages: Anaconda, Ansible, Bash, OVAL
+
+package_installed::
+* Checks if a given package is installed. Optionally, it can also check whether a specific version or newer is installed.
+* Parameters:
+** *pkgname* - name of the RPM or DEB package, eg. `tmux`
+** *evr* - Optional parameter. Use to check a specific epoch, version, release in `epoch:version-release` format. Used only in OVAL checks.
+* Languages: Anaconda, Ansible, Bash, OVAL, Puppet
+
+package_removed::
+* Checks if the given package is not installed.
+* Parameters:
+** *pkgname* - name of the RPM or DEB package, eg. `tmux`
+* Languages: Anaconda, Ansible, Bash, OVAL, Puppet
+
+sebool::
+* Checks values of SELinux booleans.
+* Parameters:
+** *seboolid* - name of SELinux boolean, eg. `cron_userdomain_transition`
+** *sebool_bool* - the value of the SELinux Boolean. Can be either `"true"` or `"false"`. If this parameter is not specified, the rule will use XCCDF Value `var_<seboolid>`
```

Comment:
Could you explain more about var_seboolid? Where is it specified, what is it? Or is it supposed that an user already knows that?

Reply:
This is actually the usual XCCDF variables that you can define in profiles, there might be some documentation available on how to use them but I'm not sure where to right now. The seboolid will be replaced by some selinux boolean, for example: selinuxuser_execheap and in the profile you can use var_selinuxuser_execheap to turn on or off the boolean.

Reply:
@jan-cerny Referencing the section which tells on how to use variables can be a good thing here.

Reply:
Do we have any section like that?

Reply:
To be honest I don't know, I just assumed there was something.